### PR TITLE
chore: Add Sarvesh Limaye

### PR DESCRIPTION
**Pull request by SarveshLimaye**
Added my profile json file

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1004"></a>

### CODE REVIEW

File: public/data/SarveshLimaye.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "url": "https://github.com/SarveshLimaye",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SarveshLimaye05",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/SarveshLimaye.json:18]: I think we should capitalize `i` here 😅 What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

Comment (SarveshLimaye) [reply public/data/SarveshLimaye.json:18]: Yeah sure , thank you for the suggestion 😄

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by SarveshLimaye** [comment]
Thank you so much for reviewing my pr @Panquesito7 😄

**Pull request merged**